src/App.js

```diff
@@ -28,7 +84,16 @@ class App extends Component {
           <Route path="/CharacterCreate" component={CharacterCreate} />
           <Route path="/CharacterList" component={CharacterList} />
           <Route path="/Dice" component={Dice} />
-          <Route path="/Settings" component={Settings} />
+          <Route
+            path="/Settings"
+            render={routeProps => (
+              <Settings
+                {...routeProps}
+                {...this.state}
+                handleToggleTheme={this.handleToggleTheme}
```

Comment:
This could be `handleToggleTheme={()=>this.handleToggleTheme()}` which would inherently bind this so we do not have to declare it in the constructor. In doing this, we can continue adding functions here for which we will not have to assign in the constructor and we can keep this cleaner.